server/src/models/Route.js

```diff
@@ -92,9 +134,19 @@ class Route extends Model {
 
   /**
    * returns route context
+   * @param {boolean} isMad true if DB schema is MAD
```

Comment:
Sorry, one last thing! (well technically two...) 

This is an example of adding the `DbContext` to JSdocs, could you please add it here and the one above :)
<img width="442" alt="Screenshot 2023-08-19 at 9 43 40 AM" src="https://github.com/WatWowMap/ReactMap/assets/58572875/e3858cb6-04d2-4599-bb61-5888001f6935">

Then I think we'll good to go!